src/main/java/uk/gov/justice/digital/hmpps/whereabouts/controllers/VideoLinkBookingController.kt

```diff
@@ -231,4 +233,16 @@ class VideoLinkBookingController(
       specification.preInterval != null
     )
   }
+
+  @PostMapping(
+    path = ["/video-link-booking-check"],
+    consumes = [MediaType.APPLICATION_JSON_VALUE],
+    produces = [MediaType.APPLICATION_JSON_VALUE]
+  )
+  fun findAvaliableVideoLinkBookingOptions(
+    @Valid
+    @RequestBody
+    specification: VideoLinkBookingSearchSpecification
```

Comment:
I guess the only consumer is your UI hence the lack of OpenApi docs around the request bodies?